samples: applications: Use ipc_radio as default radio firmware. (#15435)
nrfconnect/sdk-nrf: samples/openthread/cli/README.rst

  Not compatible with the ``tcat`` snippet.

  .. note::

>     When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the :makevar:`FILE_SUFFIX` CMake variable to ``ble``.

Reviewer: ```suggestion
    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the :makevar:`FILE_SUFFIX` CMake option to ``ble``.
    See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information.
```

>     When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :makevar:`FILE_SUFFIX` CMake variable has to be set to ``ble``.

Reviewer 2: ```suggestion
    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, set the :makevar:`FILE_SUFFIX` CMake variable to ``ble``.
```
I believe we need to say **board** target instead of build target in the new HWMv2 world, see https://github.com/nrfconnect/sdk-nrf/pull/15213

Reviewer: Yes, `build_target` is something else, as clarified with @tejlmand . Hence my PR.

>     When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` build target, the :makevar:`FILE_SUFFIX=ble` Kconfig option has to be set to ``y``.

Reviewer 2: ```suggestion
    When building with the ``multiprotocol`` snippet for the ``nrf5340dk/nrf5340/cpuapp`` board target, the :makevar:`FILE_SUFFIX=ble` Kconfig option has to be set to ``y``.
```
This is no longer a Kconfig option. How should we call it?

Reviewer: It's "CMake argument" or "CMake option", as per https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/config_and_build/configuring_app/cmake/index.html#providing-cmake-options

```
[...] you need to set the ``ble`` variant using the :makevar:`FILE_SUFFIX` argument.
```

Also, add the following line:
```
See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information.
```
